.github/CODEOWNERS

```diff
@@ -37,24 +37,24 @@
 /moveit_plugins/moveit_simple_controller_manager/	    @mikeferguson @v4hn
 /moveit_plugins/moveit_controller_manager_example/	    @v4hn
 
-/moveit_core/background_processing/                         @mlautman
-/moveit_core/backtrace/                                     @mlautman
-/moveit_core/collision_detection/                           @BryceStevenWilley
-/moveit_core/collision_detection_fcl/                       @rhaschke
-/moveit_core/collision_distance_field/                      @rhaschke
+/moveit_core/background_processing/                         @tylerjw
+/moveit_core/backtrace/                                     @tylerjw
+/moveit_core/collision_detection/                           @j-petit
+/moveit_core/collision_detection_fcl/                       @j-petit
+/moveit_core/collision_distance_field/                      @j-petit
```

Comment:
@j-petit I've made you the code owner of all things collision checking

Reply:
I don't really have a clue about the `collision_distance_field` but I can try to look into things.

Reply:
True, but that's the case for lots of code that various people are assigned to. Given your other understandings and past performance, I have a lot of faith in you :-)